x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SnapshotLifecycleTask.java

```diff
@@ -126,7 +127,7 @@ public void onFailure(Exception e) {
                         policyMetadata.getPolicy().getId(), e);
                     final long timestamp = Instant.now().toEpochMilli();
                     clusterService.submitStateUpdateTask("slm-record-failure-" + policyMetadata.getPolicy().getId(),
-                        WriteJobStatus.failure(policyMetadata.getPolicy().getId(), request.snapshot(), timestamp, e));
+                        WriteJobStatus.failure(policyMetadata.getPolicy().getId(), request.snapshot(), snapshotStartTime, timestamp, e));
```

Comment:
This start time will be `null` in this case, because `onResponse` actually sets the time, so we should remove the class var and make it local to `onResponse`